code/modules/paperwork/paper.dm

```diff
@@ -441,8 +441,8 @@
 	// Handle stamping items.
 	if(writing_stats["interaction_mode"] == MODE_STAMPING)
 		if(!user.can_read(src) || user.is_blind())
-			//The paper's stampable window area is assumed approx 400x500
-			add_stamp(writing_stats["stamp_class"], rand(0, 400), rand(0, 500), rand(0, 360), writing_stats["stamp_icon_state"])
+			//The paper's stampable window area is assumed approx 300x400
+			add_stamp(writing_stats["stamp_class"], rand(0, 300), rand(0, 400), rand(0, 360), writing_stats["stamp_icon_state"])
```

Comment:
This seems to be the proper paper dimensions. As the previous 400x500 had moments where it stamped inside an extended portion of the window.

Reply:
Also you can't use a value from 0 to 400 directly. You need to take into account the width of an stamp and subtract from this value so that the stamp isn't out of bounds of the paper.

Like if the stamp is say 50px wide then  the maximum range is only 0 to 350 so that the stamp is still fully visible inside the paper and not partially outside.  Same applies for height

Reply:
Looking at the your comment I guess this is fine too.

Reply:
I don't know / can't find the default or set values for the stamp or paper. All the dimensions, including in the fax panel, seem completely arbitary. But on local I did spam down 30 stamps and it seemed to reasonably fit the window with 300 x 400

Reply:
Fair enough, I'll trust you in this one. You can resolve this conversation